cli/src/main.rs

```diff
@@ -60,7 +60,7 @@ pub async fn main() -> ExitCode {
             }
         }
         Commands::Serve(serve_args) => {
-            install_tracing_subscriber(serve_args.verbosity());
+            install_tracing_subscriber(std::cmp::max(1, serve_args.verbosity()));
```

Comment:
I don't think this code matches your description in the commit title. This appears to limit the verbosity to 1 rather than defaulting to 1

Reply:
If I'm understanding it right, this will make it so 1 is the _minimum_ value in serve mode (if no `-v` flags are passed in, then `std::cmp::max(1, 0)` will result in 1). I am now realizing this has a different issue though where it makes the default case and the `-v` case behave the same. I think to make this truly backwards compatible with the current behavior, we'd have to do something like this:

```suggestion
            install_tracing_subscriber(std::cmp::max(1, serve_args.verbosity() + 1));
```

which I don't love tbh

Reply:
Ahh right, `max` not `min`; in that case my complaint should be "it's impossible to make the `serve` command quiet".

Is the `-v` output load-bearing in documentation or anything? I'm inclined to make this change with the expectation that folks can start using `-v` if they'd like more detail.

Alternately, could the verbosity argument be an `Option<u8>` so that we could explicitly provide a different default for the two modes if no `-v` is passed?

Reply:
> it's impossible to make the serve command quiet
Agreed that is weird, but I think that is also currently the case since the default log level is INFO

> Is the -v output load-bearing in documentation or anything?

I don't think so. I could see it maybe causing some confusion if `fastly compute serve` suddenly stopped printing INFO level stuff by default.

Reply:
> I don't think so. I could see it maybe causing some confusion if `fastly compute serve` suddenly stopped printing INFO level stuff by default.

I vote we go ahead and change it. If people have complaints we can ask the CLI folks to include a `-v` by default when `fastly compute serve` calls out to Viceroy